src/frontend/Sider/Behandling/Inngangsvilkår/Aktivitet/RegisterAktivteterTabell.tsx

```diff
@@ -35,16 +36,16 @@ const RegisterAktiviteterTabell: React.FC<{
         <Tabell size={'small'}>
             <Table.Header>
                 <Table.Row>
-                    <Table.HeaderCell scope="col">Type</Table.HeaderCell>
-                    <Table.HeaderCell scope="col">Variant</Table.HeaderCell>
-                    <Table.HeaderCell scope="col">Arrangør</Table.HeaderCell>
-                    <Table.HeaderCell scope="col">Status</Table.HeaderCell>
-                    <Table.HeaderCell scope="col">Startdato</Table.HeaderCell>
-                    <Table.HeaderCell scope="col">Sluttdato</Table.HeaderCell>
-                    <Table.HeaderCell scope="col">Aktivitetsdager</Table.HeaderCell>
-                    <Table.HeaderCell scope="col">Prosent</Table.HeaderCell>
-                    <Table.HeaderCell scope="col">Id</Table.HeaderCell>
-                    <Table.HeaderCell scope="col" />
+                    <TableHeaderCellSmall scope="col">Type</TableHeaderCellSmall>
```

Comment:
Alternativet til å lage en egen komponent for dette ville vært å skrive dette på alle rader:
```
<Table.HeaderCell textSize="small">
```